presto-docs/src/main/sphinx/functions/array.rst

```diff
@@ -21,6 +21,18 @@ The ``||`` operator is used to concatenate an array with an array or an element
 Array Functions
 ---------------
 
+.. function:: all_match(array(T), function(T,boolean)) -> boolean
+
+    Returns whether all elements of an array match the given predicate. Returns ``true`` if either all
+    the elements match the predicate or the array is empty; ``NULL`` if the predicate function returns
```

Comment:
Reword this to reflect that the second case is just a special case of the first (https://en.wikipedia.org/wiki/Vacuous_truth):  
Returns ``true`` if all elements of the array match the predicate (including when there are no elements in the array).